building/build_from_eclipse.adoc

```diff
@@ -0,0 +1,56 @@
+#  Building locally in Eclipse
+
+First, fetch the sources of the project you want to build from Github. For details, see link:configuring_git_workflow.adoc#Sources[Configuring Git Workflow - Sources].
+
+Launch Eclipse, then select `File -> Import -> Maven -> Existing Maven Projects`
+
+image::../images/import-maven-project.png[]
+
+Browse for where you cloned the repo with which you want to work.
+
+image::../images/import-maven-project-2.png[]
+
+Some projects require a full Maven build before they will compile properly in Eclipse with PDE. To solve this problem, you can either build using Maven via the commandline, or configure Eclipse to use m2e and Tycho to build the project instead of PDE. The import wizard will do this for you automatically.
+
+image::../images/import-maven-project-3-tycho.png[]
+image::../images/import-maven-project-4-tycho.png[]
+
+Note that the Tycho configurator also includes PDE, so that it can be installed even if you're using a version of Eclipse which doesn't already include it.
+
+image::../images/import-maven-project-5-tycho-pde.png[]
+
+Once the project(s) are imported, you'll want to build them. Eclipse will do this automatically with PDE, but if you want to explicitly launch Maven to perform a build, you can do so as well.
```

Comment:
I suggest adding a header here with something like "*Optional* full mvn build from eclipse" since in 99% of cases this is only needed for the site build and probably done much faster from command line than from within eclipse.